olive/passes/onnx/extract_adapters.py

```diff
@@ -35,6 +36,10 @@ class ExtractAdapters(Pass):
     If make_inputs is True, the adapter weights are inputs to the model and must be provided during inference.
     """
 
+    class SaveFormat(CaseInsensitiveEnum):
+        NPZ = "npz"
+        SAFETENSORS = "safetensors"
```

Comment:
If we move this into common/utils.py then changes need to be made to support new format will be confined to common/utils.py

Reply:
Done